ebodac/src/test/java/org/motechproject/ebodac/uitest/helper/enumBookingAppFilters.java

```diff
@@ -0,0 +1,22 @@
+package org.motechproject.ebodac.uitest.helper;
+
+public enum enumBookingAppFilters {
```

Comment:
@rubenmartin the name should be "BookingAppFilters" remove the "enum" from the name